canonical-websites/jp.ubuntu.com
# Added active state to menu based on the page you are on

## Done

Added active state to menu based on the page you are on

## QA

1. download this branch
2. run the site and go to any page other than the homepage
3. see that the nav has an active state with an orange line below the menu item you are on

## Issue / Card

Fixes #83

## Screenshots

![image](https://user-images.githubusercontent.com/441217/52207462-758d7400-2875-11e9-89d5-d4d9dc849307.png)
